netcode.h

```diff
@@ -99,6 +99,7 @@ extern "C" {
   **/
 
 #define SKILLZ_MAX_CLIENTS_PER_MATCH 2
+#define SKILLZ_MAX_MATCHES ( (int) (NETCODE_MAX_CLIENTS) / ( 2 ) )
```

Comment:
How will this round on odd numbers?

Reply:
Truncates, but I'm going to remove it.  I forgot to before.